plugins/woocommerce/src/Internal/Orders/OrderAttributionController.php

```diff
@@ -395,15 +397,15 @@ private function log( string $message, string $method, string $level = WC_Log_Le
 	 */
 	private function send_order_tracks( array $source_data, WC_Order $order ) {
 		$origin_label        = $this->get_origin_label(
-			$source_data['type'] ?? '',
+			$source_data['source_type'] ?? '',
 			$source_data['utm_source'] ?? '',
 			false
 		);
 		$customer_identifier = $order->get_customer_id() ? $order->get_customer_id() : $order->get_billing_email();
 		$customer_info       = $this->get_customer_history( $customer_identifier );
 		$tracks_data         = array(
 			'order_id'             => $order->get_id(),
-			'type'                 => $source_data['type'] ?? '',
+			'source_type'          => $source_data['source_type'] ?? '',
```

Comment:
@layoutd If we want to keep the historic tracked data consistent, we can keep the property name as before, but still use the cleaned up field name:
```suggestion
			'type'                 => $source_data['source_type'] ?? '',
```

Reply:
I think your solution is more elegant, having different terms for the same item will certainly lead to confusion in the future. Discussing it with @budzanowski, this change can be targeted for 8.6, and we can update all affected orders when the version is upgraded to avoid having to consider multiple keys.